.travis.yml

```diff
@@ -201,6 +202,7 @@ script:
   - python -m pytest python/ray/local_scheduler/test/test.py
   - python -m pytest python/ray/global_scheduler/test/test.py
 
+  - python -m pytest python/ray/test/test_global_state.py
```

Comment:
Can you add `pytest -v` in both places? (I'm adding the `-v` to all the other tests in https://github.com/ray-project/ray/pull/2420)